newsletter-november-2019.rst

```diff
@@ -0,0 +1,136 @@
+.. post:: November 04, 2019
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – November 2019
+#########################################
+
+`It's... <https://www.youtube.com/watch?v=ydyXFUmv6S4>`__ not Monty Python's Flying Circus. Sorry. It is, however, the November edition of the newsletter!
```

Comment:
Yeah, I'm gonna take this out. It might be funny to a small group of people but is probably on balance just too weird.

Reply:
well -- speaking as someone who has never really tried to get into Monty Python, I usually just go "Oh it's a Monty Python thing" and keep going.

Reply:
Yea, I've seen some bits of monty python, but don't know this reference :) No strong feeling either way though.

Reply:
I'm in favor of removing.